src/components/GeoWebContent/GWCanvas.jsx

```diff
@@ -0,0 +1,18 @@
+import React, {useEffect, useState} from 'react';
+
+import './styles.css';
+
+const GWCanvas = () =>{
+
+    let gwGateway = "https://dweb.link/ipfs/";
+    let _cid= "QmSJZ2DvrLz2hBeduYiPTsSmCXViZrbj1cc2V3mud8kX6N"
+
+    return(
+        <model-viewer className='gwCanvas' src={gwGateway+_cid} alt="Lake Boat" 
```

Comment:
Something to test out here, if a user has IPFS Companion running, will assets be loaded through their node and gateway?

Using a public gateway is great for now, but we should think more about how this is loaded long-term. Since mobile is the first use case, a public gateway is the practical option.

Reply:
Right! 
I'm not sure if in the near-future, there might be options to run a node on mobile OS itself.
Can keep that option open as well.